Client/BackEndMonitoringClient/src/CClient.cpp

```diff
@@ -148,9 +147,10 @@ bool CClient::MakeRequest(ERequestType type, std::string& message)
 	return false;
 }
 
-std::unique_ptr<CConnectorWrapper> CClient::InitConnector()
+bool CClient::InitHost(const int port, const std::string& ip_address)
```

Comment:
What is the point of such a method if it always return true

Reply:
Here I wanted to add checking, but now it is unnecessary. Deleted

Reply:
Fixed. I add method init for Host and check if is Host initialized;